seqr/views/apis/anvil_workspace_api.py

```diff
@@ -223,7 +235,7 @@ def _send_load_data_slack_msg(project, ids_path, data_path, sample_type, user):
   
         The sample IDs to load have been uploaded to {ids_path}.  
   
-        DAG for the loading pipeline:
+        {dag_name} is triggered with following:
```

Comment:
I would leave the word "DAG" in here to make it clear what was triggered, and have this be `DAG {dag_name} was triggered with following`